render.py

```diff
@@ -103,19 +115,103 @@ def display_network(net: sumolib.net.Net, stats: ET.ElementTree, max_size: int):
 
     # Flip image on the horizontal axis and update draw-pointer
     img = img.transpose(FLIP_TOP_BOTTOM)
-    draw = ImageDraw.Draw(img)
+    draw = ImageDraw.Draw(img, "RGBA")
 
-    # Draw distance legend
-    meters = find_dist_legend_size(max(city_size))
-    pixels = int(meters * width_scale)
-    draw.line([2, height - 3, 2 + pixels, height - 3], (0, 0, 0), 1)
-    draw.line([2, height, 2, height - 5], (0, 0, 0), 1)
-    draw.line([2 + pixels, height, 2 + pixels, height - 5], (0, 0, 0), 1)
-    draw.text([6, height - 18], f"{meters} m", (0, 0, 0))
+    Legend(max_size, height, draw, font) \
+        .draw_network_name(args['--net-file']) \
+        .draw_distance_legend(city_size, width_scale) \
+        .draw_gradient(((0, 255, 0), (0, 0, 255)), "Pop, work gradient") \
+        .draw_legend(COLOUR_CENTRE, "Centre") \
```

Comment:
Centre is not drawn atm, but appears on the legend.

Reply:
Fixed in 0e8eefd